scene/gui/scroll_container.cpp

```diff
@@ -350,6 +369,10 @@ void ScrollContainer::_reposition_children() {
 		fit_child_in_rect(c, r);
 	}
 
+	if (draw_focus_border) {
+		focus_panel->add_theme_style_override("panel", theme_cache.focus_style);
```

Comment:
This should happen in NOTIFICATION_THEME_CHANGED.

Reply:
`_reposition_children()` is being called conditionally on `NOTIFICATION_THEME_CHANGED`
```cpp
		case NOTIFICATION_ENTER_TREE:
		case NOTIFICATION_THEME_CHANGED:
		case NOTIFICATION_LAYOUT_DIRECTION_CHANGED:
		case NOTIFICATION_TRANSLATION_CHANGED: {
			_updating_scrollbars = true;
			callable_mp(this, is_ready() ? &ScrollContainer::_reposition_children : &ScrollContainer::_update_scrollbar_position).call_deferred();
		} break;
```
But now that I'm thinking it, I guess what you are trying to say is that doing this every time `_reposition_children()` is called is overkill and I should move those two lines to `_notification` to run only when the theme is changed

Reply:
Yes.